FROM PYTHON:3.8
RUN apt-get update && apt upgrade 
RUN
